packages/connect-query-core/tsconfig.cjs.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "./tsconfig.json",
+  "exclude": ["**/*.test.ts", "**/*.test.tsx", "vite.config.ts"]
```

Comment:
So the reason I did this was to make sure the build process validates the test.ts files. Actually, perhaps the proper solution here is to make sure `vitest` does the typechecking for the tests. This whole thing bugs me so I'm gonna take another look at it.

Reply:
I'd expect `vitest` with typechecking enabled to catch all type errors that `tsc` would catch. Seems good enough to me.

We've been using Jasmine at some places because it can run in a browser, but we're only transpiling tests to dist because there isn't a simple alternative with Jasmine.

Reply:
Yep, can confirm that vitest is doing what I want. I'll clean this up. Just fighting some weird occasional failures in test utils, `buf generate` seems to fail with a `Failure: plugin protoc-gen-es: signal: interrupt` which is weird.

Reply:
Yeah, definitely important to exclude generated code from "format" with `'!src/gen'`